gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/DagProc.java

```diff
@@ -61,5 +64,7 @@ public final void process(DagManagementStateStore dagManagementStateStore) throw
   protected abstract void sendNotification(T result, EventSubmitter eventSubmitter) throws IOException;
 
   // todo - commit the modified dags to the persistent store, maybe not required for InMem dagManagementStateStore
-  protected abstract void commit(DagManagementStateStore dagManagementStateStore, T result);
+  protected void commit(DagManagementStateStore dagManagementStateStore, T result) {
+    dagTask.conclude();
+  }
```

Comment:
sorry if I led you astray, but this impl here seems unnecessary, as `DagProcEngine` [deals directly](https://github.com/apache/gobblin/blob/de50eaf54e496a01e8a0759d942572d1359b5f98/gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagProcessingEngine.java#L98) w/ the `DagTask`.

let's remove the method entirely